Update L.DomUtil method creation to ES6 syntax

## First Time?

This is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**.

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, consider leaving this one for someone new, and looking through our general [help wanted](https://github.com/publiclab/Leaflet.DistortableImage/issues) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

## The problem

We just added support for ES6 in our library, and would like to start transitioning our code to use this syntax.

Let's update the method creation syntax in the `src/util/DomUtil.js` file from ES5 to ES6.

## Solution

**Where to find the relevant lines of code**: 

For the 5 functions in `src/util/DomUtil.js`, remove the `function` keyword and attach the method arguments to the method name instead.

Here is an example of how this will look for a different method. These ones should follow suit:

ES5
```JS
radiansToDegrees: function(angle) {
  return (angle * 180) / Math.PI;
},
```

ES6
```JS
radiansToDegrees(angle) {
  return (angle * 180) / Math.PI;
},
```

Thanks!!

## Step by Step
- [ ] Claim this issue with a comment here, below, and ask any clarifying questions you need
- [ ] Fork the repository and set it up locally following the main repo README instructions [https://github.com/publiclab/Leaflet.DistortableImage](https://github.com/publiclab/Leaflet.DistortableImage)
 - [ ] Create a new feature branch with a unique name descriptive to the issue
- [ ] Try to fix the issue following the steps above, but even before you're done, you can:
  commit your changes to your branch and start a pull request (see [contributing to Public Lab software](https://publiclab.org/wiki/contributing-to-public-lab-software)) but mark it as "in progress" if you have questions or if you haven't finished
- [ ] Reference this issue in your pull request body
- [ ] Once you submit your pull request, if there's an additional checklist provided for getting it merged, get those boxes checked off. Either way, mention me @sashadev-sky for a review.

### Please keep us updated
💬⏰ - We encourage contributors to be respectful to the community and provide an update **within a week** of claiming a first-timers-only issue. We're happy to keep it assigned to you as long as you need if you update us, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance! 

If this happens to you, don't sweat it! Grab another open issue.

##  💬   Get help
If you need any help - here are some options: 

- Comment below
- Join our gitter chat at https://gitter.im/publiclab/publiclab

Hi! I initially started working on the issue with a wrong start command... I did npm install instead of npm run set up so I had to delete the first PR and now that I seem to have got the stat right I am still getting another error without even touching the code at all...

This is what I got from npm run setup, I need help ...
`Warning: Task "karma:development:start" failed. Use --force to continue.

Aborted due to warnings.
npm ERR! code ELIFECYCLE
npm ERR! errno 3
npm ERR! leaflet-distortableimage@0.17.7 build: `cross-env NODE_ENV=production grunt build`
npm ERR! Exit status 3
npm ERR! 
npm ERR! Failed at the leaflet-distortableimage@0.17.7 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/mczenith/.npm/_logs/2020-07-18T12_25_50_719Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 3
npm ERR! leaflet-distortableimage@0.17.7 setup: `npm i && npm i leaflet --no-save && npm run build`
npm ERR! Exit status 3
npm ERR! 
npm ERR! Failed at the leaflet-distortableimage@0.17.7 setup script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/mczenith/.npm/_logs/2020-07-18T12_25_50_921Z-debug.log
`

--- a/src/util/DomUtil.js
+++ b/src/util/DomUtil.js
@@ -1,9 +1,9 @@
 L.DomUtil = L.extend(L.DomUtil, {
-  initTranslation: function(obj) {
+  initTranslation(obj) {
     this.translation = obj;
   },
 
-  getMatrixString: function(m) {
+  getMatrixString(m) {
     var is3d = L.Browser.webkit3d || L.Browser.gecko3d || L.Browser.ie3d;
 
     /*
@@ -31,17 +31,17 @@ L.DomUtil = L.extend(L.DomUtil, {
     return str;
   },
 
-  toggleClass: function(el, className) {
+  toggleClass(el, className) {
     var c = className;
     return this.hasClass(el, c) ?
       this.removeClass(el, c) : this.addClass(el, c);
   },
 
-  confirmDelete: function() {
+  confirmDelete() {
     return window.confirm(this.translation.confirmImageDelete);
   },
 
-  confirmDeletes: function(n) {
+  confirmDeletes(n) {
     if (n === 1) {
       this.confirmDelete();
       return;

